scalyr_agent/__scalyr__.py

```diff
@@ -184,18 +184,27 @@ def __add_scalyr_package_to_path():
 def __determine_version():
     """Returns the agent version number, read from the VERSION file.
     """
-    # This file can be either in the package root or the install root (if you examine the cases
-    # from above).  So, just check both locations.
-    in_install = os.path.join(get_install_root(), "VERSION")
-    in_package = os.path.join(get_package_root(), "VERSION")
-
-    if os.path.isfile(in_package):
-        version_path = in_package
-    elif os.path.isfile(in_install):
-        version_path = in_install
-    else:
+
+    file_names = ["VERSION"]
+
+    if __is_frozen__:
```

Comment:
It wouldn't hurt to add unit test for this, but I believe we should also be covered by new AMI end to end tests.

Reply:
@Kami That's a good idea. I think it would be better to start a new PR for that, to stop blocking you for now.